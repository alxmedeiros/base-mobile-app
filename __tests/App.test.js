import { render } from '@testing-library/react-native'

import App from '../App'

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = render(<App />).toJSON()
    // const tree = render.create(<App />).toJSON()
    expect(tree.children.length).toBe(1)
  })

  it('renders correctly', () => {
    // const tree = render.create(<App />).toJSON()
    const tree = render(<App />)
    expect(tree).toMatchSnapshot()
  })
})
