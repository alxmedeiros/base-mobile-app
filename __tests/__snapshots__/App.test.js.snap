// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#ff0000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
</View>
`;
